src/services/jsDoc.ts

```diff
@@ -315,9 +316,14 @@ namespace ts.JsDoc {
         }).join("");
     }
 
+    function returnsDocComment(isJavaScriptFile: boolean, indentationStr: string, newLine: string) {
+        return `${indentationStr} * @returns ${isJavaScriptFile ? "{any}" : ""}${newLine}`;

```

Comment:
We should have a test for `@returns {any}` in JS files if we add it.

Reply:
@jessetrinity Done.